### PR TITLE
Add windows and mac deployment with bun

### PR DESCRIPTION
**Pull request by patrickelectric**

### CODE REVIEW

**Comment by patrickelectric** [comment]
@joaoantoniocardoso it appears that the mac build is being limited by the organization limit.

File: .github/workflows/deploy-unix.yml [hunk 1]
```diff
@@ -21,11 +21,21 @@ jobs:
             TARGET: aarch64-unknown-linux-gnu
             SUFFIX: aarch64
 
+          - os: macos-13-xlarge
+            TARGET: aarch64-apple-darwin
+            SUFFIX: macos-aarch64
+
+          - os: windows-latest
```
Comment (joaoantoniocardoso) [inline .github/workflows/deploy-unix.yml:28]:
```suggestion
          - os: windows-2022
```

File: .github/workflows/deploy-unix.yml [hunk 2]
```diff
@@ -21,11 +21,21 @@ jobs:
             TARGET: aarch64-unknown-linux-gnu
             SUFFIX: aarch64
 
+          - os: macos-13-xlarge
```
Comment (joaoantoniocardoso) [inline .github/workflows/deploy-unix.yml:24]:
```suggestion
          - os: macos-13
```

Comment (joaoantoniocardoso) [reply .github/workflows/deploy-unix.yml:24]: large runners require enterprise stuff

File: .github/workflows/deploy-unix.yml (hunk 1 above)
Comment (joaoantoniocardoso) [reply .github/workflows/deploy-unix.yml:28]: `latest` is always out of out control, then it updates and the build suddenly starts breaking

File: build.rs [hunk 3]
```diff
@@ -82,49 +99,49 @@ fn generate_typescript_bindings() {
...
         .status()
         .unwrap();
 
     #[cfg(debug_assertions)]
-    let install = Command::new("yarn")
+    let install = Command::new("bun")
         .args(["install", "--frozen-lockfile"])
         .current_dir(frontend_dir)
         .status()
         .unwrap();
```
Comment (joaoantoniocardoso) [inline build.rs:133]: why do we have these two of the same? haha

**Comment by patrickelectric** [comment]
@joaoantoniocardoso updated

**Review by joaoantoniocardoso — APPROVED** [review]

**Pull request merged**